sessionctx/variable/session.go

```diff
@@ -444,6 +446,26 @@ func (s *SessionVars) ResetPrevAffectedRows() {
 	}
 }
 
+// GetExecuteArgumentsInfo gets the argument list as a string of execute statement.
+func (s *SessionVars) GetExecuteArgumentsInfo() string {
+	if len(s.PreparedParams) == 0 {
+		return ""
+	}
+	args := make([]string, 0, len(s.PreparedParams))
+	for _, v := range s.PreparedParams {
+		if v.IsNull() {
+			args = append(args, "<nil>")
+		} else {
+			str, err := v.ToString()
+			if err != nil {
+				terror.Log(err)
```

Comment:
Add some details to this error log.

Reply:
It has some details about it. Actually, it should not meet the error, just make the errcheck happy.